src/background/modules/UnicodeFontHandler.js

```diff
@@ -207,6 +207,7 @@ function applySettings(unicodeFont) {
  * @returns {void}
  */
 export async function init() {
+	// Remove once https://bugzilla.mozilla.org/show_bug.cgi?id=1595822 is fixed
```

Comment:
Hmm, this comment comments a `await isMobile()` exclusion, but your bug is about Thunderbird?
How is this related?

`isMobile` AFAIK should only check (and in this case exclude) Firefox for Android…

Reply:
No, that bug is about implementing the WebExtension context menu API in Firefox for Android.